README.md

```diff
@@ -1,151 +1,185 @@
+
+data "ibm_container_cluster_config" "cluster_config" {
+  cluster_name_id = "xxxxxxxxx" # replace with cluster ID or name
+}
 
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
+# ############################################################################
+# Config providers
+# ############################################################################
 
-```hcl
-terraform {
-  required_version = ">= 1.9.0"
-  required_providers {
-    ibm = {
-      source  = "IBM-Cloud/ibm"
-      version = "X.Y.Z"  # Lock into a provider version that satisfies the module constraints
-    }
-  }
+provider "ibm" {
+  ibmcloud_api_key = "xxxxxxxxxxxx"  # pragma: allowlist secret
 }
 
-locals {
-    region = "us-south"
+provider "helm" {
+  kubernetes {
+    host                   = data.ibm_container_cluster_config.cluster_config.host
+    token                  = data.ibm_container_cluster_config.cluster_config.token
+    cluster_ca_certificate = data.ibm_container_cluster_config.cluster_config.ca_certificate
+  }
+  registries = [
+    { url = "oci://icr.io", username = "iamapikey", password = var.ibmcloud_api_key }
+  ]
 }
 
-provider "ibm" {
-  ibmcloud_api_key = "XXXXXXXXXX"  # replace with apikey value
-  region           = local.region
+provider "kubernetes" {
+  host                   = data.ibm_container_cluster_config.cluster_config.host
+  token                  = data.ibm_container_cluster_config.cluster_config.token
+  cluster_ca_certificate = data.ibm_container_cluster_config.cluster_config.ca_certificate
 }
 
-module "module_template" {
-  source            = "terraform-ibm-modules/<replace>/ibm"
-  version           = "X.Y.Z" # Replace "X.Y.Z" with a release version to lock into a specific release
-  region            = local.region
-  name              = "instance-name"
-  resource_group_id = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX" # Replace with the actual ID of resource group to use
+# ############################################################################
+# Install DSC
+# ############################################################################
+module "backup_recovery" {
+  source  = "terraform-ibm-modules/iks-ocp-backup-recovery/ibm"
+  version = "X.Y.Z"  # Replace "X.Y.Z" with a release version to lock into a specific release
+  cluster_id                    = "xxxxxxx" # replace with ID of the cluster
+  cluster_resource_group_id     = "xxxxxxx" # replace with ID of the cluster resource group
+  dsc_registration_token        = "xxxxxxx" # replace with Registration_token
+  connection_id                 = "xxxxxxx" # replace with connection ID
+  # --- B&R Instance ---
+  brs_instance_guid   = "xxxxxxx" # replace with ID of the BRS instance
+  brs_instance_region = var.region
+  brs_endpoint_type   = "public"
+  brs_tenant_id       = "xxxxxxx" # replace with tenant ID of the BRS instance
+  registration_name   = "ocp-dsc"
+  registration_images = {
```

Comment:
I would remove this from this usage because you now added default values, so consumers don't need to explicially pass values. Keep the usage block simple and only include the required inputs